recipes/onnxruntime/all/conanfile.py

```diff
@@ -0,0 +1,227 @@
+import os
+from conans import ConanFile, tools
+
+
+class OnnxRuntimeConan(ConanFile):
+    name = "onnxruntime"
+    description = "ONNX Runtime: cross-platform, high performance scoring " \
+                  "engine for ML models https://aka.ms/onnxruntime"
+    license = "MIT License"
+    topics = (
+        "deep-learning",
+        "onnx",
+        "neural-networks",
+        "machine-learning",
+        "ai-framework",
+        "hardware-acceleration"
+    )
+    homepage = "https://www.onnxruntime.ai/"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "compiler", "build_type", "arch", "os_build"
+
+    options = {
+        "shared": [True, False],
+        "parallel": [True, False],
+        "force_min_size_rel": [True, False],
+        "with_dml": [True, False],
+        "with_openmp": [True, False],
+        "with_dnnl": [True, False],
+        "with_nnapi": [True, False],
+        "with_winml": [True, False],
+        "with_python_wheel": [True, False],
+        "with_csharp": [True, False],
+        "with_java": [True, False],
+        "with_tests": [True, False]
+    }
+
+    default_options = {k: False for k, v in options.items()}
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def build_requirements(self):
+        self.build_requires("protobuf/3.11.3")
+
+    def source(self):
+        # Unfortunatelly git do not pack submodules, so code below doesn't work
+        #
+        # tools.get(**self.conan_data["sources"][self.version])
+        # os.rename(f"onnxruntime-{self.version}", self._source_subfolder)
+        #
+        # Using tools.Git to get it:
+        git = tools.Git(folder=self._source_subfolder)
+        git.clone("https://github.com/microsoft/onnxruntime.git")
+        git.checkout(f"v{self.version}", submodule="recursive")
+
+    def build(self):
+        is_windows = self.settings.os_build == "Windows"
+        build_script = ".\\build.bat" if is_windows else "./build.sh"
+        build_args = ["--skip_submodule_sync"]
```

Comment:
don't do that please, call upstream CMakeLists through CMake wrapper, and set options to use external libraries and not vendored one: https://github.com/microsoft/onnxruntime/blob/6334c292408a1df4d19835a1edd1d563d932880b/cmake/CMakeLists.txt#L107-L111

Reply:
Done